shared/constants/transaction.js

```diff
@@ -183,6 +191,7 @@ export const TRANSACTION_GROUP_CATEGORIES = {
  *  ready to submit to the network.
  * @property {string} hash - A hex string of the transaction hash, used to
  *  identify the transaction on the network.
- * @property {number} submittedTime - The time the transaction was submitted to
+ * @property {number} [submittedTime] - The time the transaction was submitted to
  *  the network, in Unix epoch time (ms).
+ * @property {TxError} [err] - The error encountered during the transaction
```

Comment:
Awww :disappointed: Well, today I learned that this is a thing that we do. We should be preserving the `Error` object, not deconstructing it into this "TxError" object. Oh well. A task for a future PR.